userguide/services.rst

```diff
@@ -1484,7 +1484,7 @@ This configuration screen is really a front-end to
    | SMB2_10        | used by Windows 7                                          |
    |                |                                                            |
    +----------------+------------------------------------------------------------+
-   | SMB3           | used by Windows 8                                          |
+   | SMB3           | Same as SMB3_11                                            |
```

Comment:
I see your point. Since SMB3 = SMB3_11, "used by Windows 8" is technically incorrect. This is relevant when setting server minimum protocol = SMB3.  
Does the following look better?
**used by Windows 10; same as SMB3_11**